### PR TITLE
Fix tables in comments with mixed whitespace

### PR DESCRIPTION
**Pull request by YoshiRulz**
Someone's IDE put smart tabs in comments:
pic

### CODE REVIEW

**Comment by zeromus** [comment]
disch notes are for implementers not API -- doesnt belong in doc comments
TODOs are for implementers, not API -- doesnt belong in doc comments.

**Comment by YoshiRulz** [comment]
If you're really concerned about comment visibility, I can easily move the comment in NES-EVENT.cs out of `<summary />`. If I'm going to touch TODOs, though, it'll be [all at once](https://i.imgur.com/RlMFdFA.png).

**Comment by zeromus** [comment]
things not to do in someone else's house if you want them to invite you back:
1. gently re-adjust pictures which are hanging kind of crooked
2. move items from the trash to the recycling
3. reorganize their sock drawer

**Comment by adelikat** [comment]
I agree, don't change it to summary comments for the mapper comments.  If those can be put back this is good to go

**Comment by YoshiRulz** [comment]
Should it be in `<remarks />` (which I've since learned about) or a multi-line comment?

**Comment by vadosnaprimer** [comment]
What if this is done via blaming and undoing whatever screwed up the alignments in the first place?

File: BizHawk.Emulation.Cores/Consoles/Atari/2600/Mappers/mAR.cs [hunk 1]
```diff
@@ -2,35 +2,17 @@
 using System.Linq;
 
 using BizHawk.Common;
-/**
```
Comment (adelikat) [inline BizHawk.Emulation.Cores/Consoles/Atari/2600/Mappers/mAR.cs]: Revert this

File: BizHawk.Emulation.Cores/Consoles/Nintendo/NES/Boards/NES-EVENT.cs [hunk 2]
```diff
@@ -1,122 +1,116 @@
-﻿#region Disch's Notes
-/*
```
Comment (adelikat) [inline BizHawk.Emulation.Cores/Consoles/Nintendo/NES/Boards/NES-EVENT.cs]: Revert this

File: BizHawk.Emulation.Cores/Consoles/Sega/SMS/SMS.cs [hunk 3]
```diff
@@ -7,14 +7,12 @@
...
 
-/*****************************************************
-  TODO: 
-  + HCounter (Manually set for light phaser emulation... should be only case it's polled)
-  + Try to clean up the organization of the source code. 
-  + Mode 1 not implemented in VDP TMS modes. (I dont have a test case in SG1000 or Coleco)
- 
-**********************************************************/
-
+/// <summary>
```
Comment (adelikat) [inline BizHawk.Emulation.Cores/Consoles/Sega/SMS/SMS.cs]: Revert this

**Comment by vadosnaprimer** [comment]
LGTM

**Pull request merged**